.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Erlang Tests CI
+on: push
+
+jobs:
+  Run:
+    strategy:
+      matrix:
+        erlang: ['24']
+    runs-on: ubuntu-latest
+    container:
+      image: erlang:${{ matrix.erlang }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        env:
+          cache-name: rebar3
+        with:
+          path: |
+            _build
+            ~/.cache/rebar3
+          key: ci-${{runner.os}}-${{env.cache-name}}-erlang_${{matrix.erlang}}-rebar3-${{hashFiles('rebar.lock')}}
+          restore-keys: |
+            ci-${{runner.os}}-${{env.cache-name}}-erlang_${{matrix.erlang}}-rebar3
+            ci-${{runner.os}}-${{env.cache-name}}-erlang_${{matrix.erlang}}
```

Comment:
Same thing as the other PR about the `restore-keys` cache management with upgrading rebar dependencies.

```suggestion
```